catris.service

```diff
@@ -16,6 +16,7 @@ Description=catris
 [Service]
 ExecStart=/home/catris/catris
 Environment="RUST_BACKTRACE=1"
+Environment="CATRIS_WEBSOCKET_PROXIED=1"
```

Comment:
This could be an IP address, in case proxy and rust program are on different servers.